jflex/src/test/java/jflex/core/IntCharSetProperties.java

```diff
@@ -0,0 +1,171 @@
+/* * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * *
+ * JFlex 1.8.0-SNAPSHOT                                                    *
+ * Copyright (C) 1998-2019  Gerwin Klein <lsf@jflex.de>                    *
+ * All rights reserved.                                                    *
+ *                                                                         *
+ * License: BSD                                                            *
+ *                                                                         *
+ * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * */
+
+package jflex.core;
+
+import static com.google.common.truth.Truth.assertThat;
+
+import com.pholser.junit.quickcheck.Property;
+import com.pholser.junit.quickcheck.generator.InRange;
+import com.pholser.junit.quickcheck.runner.JUnitQuickcheck;
+import org.junit.runner.RunWith;
+
+/**
+ * Property-based tests for {@link IntCharSet}
+ *
+ * @author Gerwin Klein
+ * @version JFlex 1.8.0-SNAPSHOT
+ * @see IntCharSet
+ */
+@RunWith(JUnitQuickcheck.class)
+public class IntCharSetProperties {
```

Comment:
This file doesn't contain define properties ; I think we should just name in IntCharSetQuickcheck.

Reply:
Ah, clash of terminology. The things that quickcheck tests are called properties there (because they are supposed to resemble logical properties that you write for all inputs), and they all declare `@Property`. I can see how this is ambiguous wrt to Java properties, though. Happy with `.*QuickCheck`.